docker-compose.yaml

```diff
@@ -0,0 +1,77 @@
+version: '3.5'
+services:
+  redis:
+    image: redis:6.0.6-alpine3.12
+    volumes:
+      - redis-data:/data
+    ports:
+    - "${REDIS_PORT:-6379}:6379"
+  test:
+    depends_on:
+      - redis
+    profiles:
+      - test
+    command: ["sleep" ,"9999"]
+    environment:
+      SYMWS_URL: 'https://cat-stage.libraries.psu.edu:28443/symwsbc'
+      SYMWS_USERNAME: 'myaccount'
+      REDIS_SIDEKIQ_URI: "redis://redis:6379/1"
+      RAILS_ENV: "test"
+      REDIS_DATABASE_URI: "redis://redis:6379/2"
+      SYMWS_PIN: "${SYMWS_PIN}"
+    tty: true
+    stdin_open: true
+    build:
+      context: .
+      target: test
+    volumes:
+    - bundle-data:/app/vendor/bundle
+    - node-data:/app/node_modules
+    - type: bind
+      source: ./
+      target: /app/
+  sidekiq:
+    environment:
+      SYMWS_URL: 'https://cat-stage.libraries.psu.edu:28443/symwsbc'
+      SYMWS_USERNAME: 'myaccount'
+      REDIS_SIDEKIQ_URI: "redis://redis:6379/1"
+      RAILS_ENV: "development"
+      REDIS_DATABASE_URI: "redis://redis:6379/2"
+      SYMWS_PIN: "${SYMWS_PIN}"
+    tty: true
+    stdin_open: true
+    build:
+      context: .
+      target: base
+    volumes:
+    - bundle-data:/app/vendor/bundle
+    - node-data:/app/node_modules
+    - type: bind
+      source: ./
+      target: /app/
+    command: ["/app/bin/sidekiq"]
+  web:
+    environment:
+      SYMWS_URL: 'https://cat-stage.libraries.psu.edu:28443/symwsbc'
+      SYMWS_USERNAME: 'myaccount'
+      REDIS_SIDEKIQ_URI: "redis://redis:6379/1"
+      RAILS_ENV: "development"
+      REDIS_DATABASE_URI: "redis://redis:6379/2"
+      SYMWS_PIN: "${SYMWS_PIN}"
+    tty: true
+    stdin_open: true
+    build:
+      context: .
+      target: base
+    volumes:
+    - bundle-data:/app/vendor/bundle
+    - node-data:/app/node_modules
+    - type: bind
+      source: ./
+      target: /app/
+    ports:
+      - "3000:3000"
```

Comment:
@whereismyjetpack would it be helpful to have this one configurable as well? we can just change it here but maybe makes more sense to do it in `.envrc` since it is not tracked?

Reply:
Yeah, we can do that. I think I'll also put examples in the .sample file

Reply:
I've added config bits for the app port, as well.